packages/identity/src/sign.ts

```diff
@@ -11,3 +11,12 @@ export const sign = async (message: string, privKey: CryptoKey): Promise<ArrayBu
 
   return await crypto.sign(algorithm.algorithm as Algorithm, privKey, messageBuffer)
 }
+
+export const signData = async (data: Uint8Array, privKey: CryptoKey): Promise<ArrayBuffer> => {
```

Comment:
It wouldn't be necessary to duplicate method if we extended data type
```
data: string | Uint8Array
```

Reply:
Same comment above.